app/models.py

```diff
@@ -905,18 +905,20 @@ def apply(self, domain, post_records):
                         if temp_content.strip()[-1:] != '.':
                             temp_content += '.'
 
+                    if new_record['ttl'] is None:
+                        new_record['ttl'] = item['ttl']
```

Comment:
Set TTL to the first TTL of that name+type tuple (which is the record sorted topmost in the GUI).